install-paperless-ngx.sh

```diff
@@ -361,6 +376,10 @@ if [[ -n $POSTGRES_FOLDER ]] ; then
 	sed -i "/^\s*pgdata:/d" docker-compose.yml
 fi
 
+if [[ -n $MARIADB_FOLDER ]] ; then
+	sed -i "s#- dbdata:/var/lib/mariadb/data#- $MARIADB_FOLDER:/var/lib/mariadb/data#g" docker-compose.yml
```

Comment:
This still uses `/var/lib/mariadb/data` instead of `/var/lib/mysql`.  But I think with a single `dbdata` volume, it wouldn't be needed?